src/Common/State.php

```diff
@@ -72,6 +73,13 @@ final class State
      */
     private $CityCode;
 
+    /**
+     * @JMS\Type("bool")
+     *
+     * @var bool
+     */
+    private $finel = false;
```

Comment:
```suggestion
    const STATUS_DELIVERED = 4; // Успешно доставлен и вручен адресату (конечный статус)
    const STATUS_NOT_DELIVERED = 5; // Не вручен; Покупатель отказался от покупки, возврат в ИМ (конечный статус)
```
(Комменты бы как PHPDoc оформить)